release/7-1/arm32v7/docker/Dockerfile

```
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation. 
 # Licensed under the MIT License.
 
-FROM arm32v7/ubuntu:bionic AS installer-env
+FROM arm32v7/ubuntu:focal AS installer-env
```

We should pull focal from the official repo
Also, it would be good if we kept the bionic image for now.
```suggestion
FROM --platform=linux/arm/v7 ubuntu:focal
```